plugin-server/src/cdp/templates/index.ts

```diff
@@ -91,4 +91,5 @@ export const HOG_FUNCTION_TEMPLATES: HogFunctionTemplate[] = [
     ...HOG_FUNCTION_TEMPLATES_SOURCES,
     ...HOG_FUNCTION_TEMPLATES_COMING_SOON,
     ...(NATIVE_HOG_FUNCTIONS as unknown as HogFunctionTemplate[]),
+    ...(NATIVE_HOG_FUNCTIONS as unknown as HogFunctionTemplate[]),
```

Comment:
Not sure why this is here also the cast is really nasty. This actually caught me in a different PR... Can you rework this to be like the Plugin ones please so that we have strong types

Reply:
will fix the type in a followup pr